pudb/__init__.py

```diff
@@ -106,8 +106,8 @@ def runmodule(*args, **kwargs):
 
 
 def runscript(mainpyfile, args=None, pre_run="", steal_output=False,
-              run_as_module=False):
-    dbg = _get_debugger(steal_output=steal_output)
+              _continue=False, run_as_module=False):
```

Comment:
I think it'd be nicer if these were also called `_continue_at_start` or so.

Reply:
Fair point! I had to come up with a different name for the permanent value, it came out a bit awkward but hopefully it's clear at least. I wonder if it would make sense to capture that sort of thing in a settings object instead though.

Reply:
Awkward for sure, but I've seen worse. This works for me.